resources/todomvc/architecture-examples/vue/src/components/TodosComponent.vue

```diff
@@ -17,36 +13,38 @@
                     :class="{ completed: todo.completed, editing: todo === editing }">
                     <div class="view">
                         <input type="checkbox" v-model="todo.completed" class="toggle">
-                        <label @dblclick="editTodo(todo)">{{ todo.title }}</label>
+                        <label @dblclick="startEdit(todo)">{{ todo.title }}</label>
                         <button class="destroy" @click.prevent="deleteTodo(todo)"></button>
                     </div>
-                    <input type="text" class="edit" v-model="todo.title" @keyup.enter="doneEdit" @blur="doneEdit"
-                        v-todoFocus="todo === editing" />
+                    <input type="text" class="edit" v-model="todo.title" @keyup.enter="finishEdit" @blur="finishEdit" v-todoFocus="todo === editing" />
                 </li>
```

Comment:
Ah I just see your line in the PR description and the follow-up discussion!
I'd be happy to see a PR with just this change, so that we can study it in isolation.